config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -28,22 +28,24 @@
         "4.3.0": {
             "status": "deprecated",
             "latest-revision": {
+                "revision": "12",
+                "release-date": "09 January 2023",
+                "cves": "CVE-2022-42898",
```

Comment:
`"cves": "CVE-2022-42898",` ----->
`"cves": "CVE-2022-42898,CVE-2022-41717,CVE-2022-41720",`

and 

`"fixes": [
                    "Fixes CVE-2022-42898,CVE-2022-41717,CVE-2022-41720 and PSIRT PVR0406812,PVR0410879,PVR0410611"
                 ]`
-----> `"fixes": []`